Sources/Internal/FrameProvider.swift

```diff
@@ -152,6 +163,16 @@ final class FrameProvider {
     return CGRect(origin: CGPoint(x: x, y: y), size: daySize)
   }
 
+  func frameOfMonthFooter(_ month: Month, inMonthWithOrigin monthOrigin: CGPoint) -> CGRect {
```

Comment:
we can omit the `month` parameter here. I don't believe it's used.

Reply:
Month is used here to determine how many rows for the month are being displayed

Reply:
Ah! I missed that every time I looked, whoops!